### DISCUSSION TITLE
A random question about the functionality of xdg-desktop-portal-hyprland!

### DISCUSSION THREAD

**Discussion by ghost** [hyprwm/Hyprland#8760] [General]
While utilizing xdg-desktop-portal-hyprland, I have experienced random freezing and the loss of some recordings. I'm not sure if this is related to my video recorder or the portal itself. However, I have been trying the xdg-desktop-portal-wlr, and it seems to be working (though I need to test more before being certain). My question is: I'm getting the impression that the audio worked better with the Hyprland portal compared to the WLR portal. Is there any possibility that the portals could affect the audio in any way? Or am I just imagining it, and the portals shouldn't affect the audio at all?

Please answer only if you have a complete understanding about the functionality of the portal.

**Comment by vaxerski** [comment]
I don't think the portal should affect audio in any way unless it's somehow lagging pipewire itself (unlikely)
